test/func/fixtures/jquery.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Fixture</title>
+  </head>
+  <body>
+    <div class="e2e-content"></div>
+    <div class="e2e-error"></div>
+    <script src="error-handler.js"></script>
+    <script src="../../../lib/little-loader.js"></script>
+    <script src="jquery/jquery.js"></script>
```

Comment:
Ah gotcha! I'm gonna interpret it wrong every time I see it, maybe rename to `jquery-test` or `load-jquery`?